### ISSUE TITLE
Offline Cache Missing CO and Hidden Date

### ISSUE THREAD

**Issue by cachly** [cachly/bug-tracker#5008]
**[Original report](https://bitbucket.org/zedsaidstudio/public-issues-tracker/issues/148) by Carson Monroe (Bitbucket: [ACME_WildCachers](https://bitbucket.org/ACME_WildCachers), ).**

----------------------------------------

With 3.1 beta 4, the cache details screen is missing the line with the CO and date hidden. 

![IMG_6251.PNG](https://bitbucket.org/repo/pkzxL5/images/180584154-IMG_6251.PNG)

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

Need more info. Nothing was changed that would affect this in beta 4. Is this a PQ again? Is this a cache you had from a previous beta? How was this cache saved offline?

**Comment by cachly** [comment]
**Original comment by Carson Monroe (Bitbucket: [ACME_WildCachers](https://bitbucket.org/ACME_WildCachers), ).**

----------------------------------------

I have a PQ for unfound, available caches near home that runs on Friday mornings. This cache would have been in that PQ starting last week on 8/18 when I was on 3.1 beta 3. This morning i am running 3.1 beta 4. I went to the existing list that had this cache on it (the only list that contains this cache) and removed all caches. Then I downloaded the latest PQ to a new list, and merged that list into the original.

I do not know if I explicitly viewed this cache from the offline list prior to this morning, so can't say for certain that it's a new problem. But I do know that this is not limited to one cache, which makes it less likely that it existed prior to the most recent beta. Six out of the first ten caches on the list view of the list showed the same issue. I have not been able to discern any particular pattern as to which caches are affected and which are not. Caches that I know are in other lists sometimes have this issue and sometimes do not. But the problem is consistent across lists containing a cache, so it seems this info has been wiped out on the single database record  for the affected caches. I was trying to see if I could come up with a filter to restrict to only affected caches so I could take a more targeted look, but it looks like you can't leave owner name blank and there are no logical operators for "does not contain" or similar exclusions (would be a good enhancement suggestion).

Let me know if I can provide any more info to help track this down.

**Comment by cachly** [comment]
**Original comment by Carson Monroe (Bitbucket: [ACME_WildCachers](https://bitbucket.org/ACME_WildCachers), ).**

----------------------------------------

I have not been keeping up with the forums, but have been trying to catch up over the last week or two. I noticed this sounds very much like the issue discussed in a [misnamed thread](http://www.cach.ly/support/index.php?/topic/615-found-filter/) over there, though the timing doesn't really line up. I have not seen the problem since I reported it and reproduced it the same day. I don't have any specific reason to believe this was different other than the timing, so feel free to close.
